concent_api/core/views.py

```diff
@@ -347,13 +369,12 @@ def receive_out_of_band(_request, _message):
             last_undelivered_receive_out_of_band_status.save()
 
             message_verdict.sig = None
-            golem_message, message_timestamp = store_message(
+            (golem_message, message_timestamp) = store_message(
                 message_verdict.TYPE,
                 decoded_last_task_message.message_cannot_compute_task.task_to_compute.compute_task_def['task_id'],
                 message_verdict.serialize()
             )
             store_receive_out_of_band(golem_message, message_timestamp)
-            message_verdict.sig = None
```

Comment:
OK. Now why are you **not** removing the sig here? I mean, I was just asking because it's a change out of nowhere. You should have such changes in separate commits and properly described so that we know exactly why we're changing it instead of changing the code randomly.